src/screens/sign-up/sign-up-screen.tsx

```diff
@@ -1,12 +1,160 @@
 import * as React from "react";
-import { Text, View } from "react-native";
-import { COMMON_STYLES } from "~theme/common-styles";
+import { useEffect, useState } from "react";
+import {
+  Text,
+  TouchableWithoutFeedback,
+  View,
+  Keyboard,
+  ScrollView,
+} from "react-native";
+import { COMMON_STYLES } from "../../theme/common-styles";
+import ButtonComponent from "./components/ButtonComponent";
+import FormFieldsComponent from "./components/FormFieldsComponent";
+import LogoComponent from "./components/LogoComponent";
+import axios from "axios";
+import Icon from "react-native-vector-icons/FontAwesome5";
+
+export interface UserDetailsProps {
+  firstName?: string;
+  lastName?: string;
+  fullName?: string;
+  email?: string;
+  phoneNumber?: string;
+  password?: string;
+  [key: string]: any;
+}
+
+export enum Position {
+  ABSOLUTE = "absolute",
+  RELATIVE = "relative",
+}
+
+interface StyleProps {
```

Comment:
It will be nice to define your interfaces, enums and types in distinct files. Your code will be smaller and cleaner that way

Reply:
Created a folder named types and created two files: interfaces and enums inside this types folder where I stored all interfaces and enums respectively.